test/hotspot/gtest/memory/test_virtualspace.cpp

```diff
@@ -59,7 +59,7 @@ namespace {
   static void test_reserved_size(size_t size) {
     ASSERT_PRED2(is_size_aligned, size, os::vm_allocation_granularity());
 
-    ReservedSpace rs = MemoryReserver::reserve(size, mtTest);
+    ReservedSpace rs = MemoryReserver::reserve(size, mtNone);
```

Comment:
Why did you do this change?
```suggestion
    ReservedSpace rs = MemoryReserver::reserve(size, mtTest);
```